src/main/resources/templates/targets/aws-cloudformed-s3-target-template.yaml

```diff
@@ -47,6 +47,21 @@ target:
           DeliveryLBDomainName: ${aws.cloudformation.deliveryLBDomainName}
           CloudFrontCertificateArn: ${aws.cloudformation.cloudfrontCertificateArn}
           AlternateCloudFrontDomainNames: ${aws.cloudformation.alternateCloudFrontDomainNames}
+    {{#if source_target}}
+    duplicate:
+      - hookName: duplicateIndexLifecycleHook
+      - hookName: duplicateProcessCommitsFileLifecycleHook
+      - hookName: duplicateS3LifecycleHook
+        region: ${aws.region}
+        accessKey: ${aws.accessKey}
+        secretKey: ${aws.secretKey}
+        url: {{aws.s3.url}}
+        endpoint: {{aws.s3.endpoint}}
+        pathStyleAccess: {{aws.s3.path_style_access}}
+        sourceConfig:
+          url: {{source.aws.s3.url}}
+          localRepoPath: {{source.local_repo_path}}
```

Comment:
Can't remember if we agreed to add blob store duplication as part of the duplicate hooks. If we agreed to not do it, this is fine.

Reply:
Studio owns the blob-stores. It is the one writing to it on content-write/publish. It should be Studio handling the duplicate as well